DTS-STN/Alpha-Site
# Content changes as per Figma

# Description

[#579 Content changes in Figma](https://trello.com/c/E9pKG2bY)

Modified content on the experiments, about and sign up pages as per the changes in Figma.

## Acceptance Criteria

Done when the content matches what is in Figma

## Test Instructions

1. Pull in branch
2. Type `npm run dev`
3. Navigate to each page to ensure the content is correct


## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
